examples/nats-pub/__main__.py

```diff
@@ -43,7 +43,7 @@ async def run():
     # e.g. nats-pub -s demo.nats.io hello "world"
     parser.add_argument('subject', default='hello', nargs='?')
     parser.add_argument('-d', '--data', default="hello world")
-    parser.add_argument('-s', '--servers', default="")
+    parser.add_argument('-s', '--server', default="")
```

Comment:
This example can consume only one server at time.

Reply:
You are right, the client can handle more than one server, so I decided to revert this part of my changes.
Also changing this name would require changing the doc outside the scope of this repository, so I prefer to leave it as it is now.
https://docs.nats.io/running-a-nats-service/nats_docker/ngs-docker-python